ssl/os_port.h

```diff
@@ -98,6 +108,17 @@ extern "C" {
 
 void ax_wdt_feed();
 
+#ifndef PROGMEM
+#define PROGMEM __attribute__((aligned(4))) __attribute__((section(".irom.text")))
+#endif
+
+
+#ifndef FORCEL32_NOT_SUPPORTED
+#define flashArrayRead(x,y) x[y]
```

Comment:
Since the concept of storing arrays in flash is exposed in the code using the `PROGMEM` attribute, suggest renaming this to `progmemArrayRead` to make things more obvious for the reader.